### PR TITLE
Upgrade to November's Closure Compiler

### PR DESCRIPTION
**Pull request by kristoferbaxter**
The issue with AMP using the latest compiler is due to broken/incorrect sourcemaps within the amphtml project.

Bumping version after local testing with a patch to be submitted to `amphtml`.

### CODE REVIEW

File: .github/workflows/build-test-push.yml
Comment (rsimha) [inline .github/workflows/build-test-push.yml:10]: I think this change should be reverted. The intent of the conditional is to prevent yet another regeneration of binaries after [these steps](https://github.com/ampproject/amp-closure-compiler/blob/master/PUBLISH.md#publishing-a-new-version-of-ampprojectamp-closure-compiler) have been run to update the version.

**Review by rsimha — APPROVED** [review]
LGTM after reverting the yml change.

File: .github/workflows/build-test-push.yml
Comment (kristoferbaxter) [reply .github/workflows/build-test-push.yml:10]: Without commenting this line out, the Github Actions fail to start on this PR. Perhaps there is a tweak that satisfies both conditions?

**Comment by rsimha** [comment]
Commenting from the mobile app, so can't confirm, but I'm pretty sure you can replace all the build instructions with a call to `node tasks/build.js`.

File: .github/workflows/build-test-push.yml
Comment (rsimha) [reply .github/workflows/build-test-push.yml]: That seems unexpected. Is this a conventional PR from a fork?

Comment (kristoferbaxter) [reply .github/workflows/build-test-push.yml]: This PR is a bump of the version number for `closure-compiler` directly (not from renovate).

When building locally, everything works and symlinking (via npm link) works as expected with `amphtml`.

Comment (rsimha) [reply .github/workflows/build-test-push.yml]: I now see what you're seeing.

image

I'm happy to fix this via a separate PR. Meanwhile, feel free to comment out this line and make progress.

We'll need to manually cancel the jobs that come up during publishing, I can do that for you once this PR is merged.

Comment (rsimha) [reply .github/workflows/build-test-push.yml]: I just pushed a commit to this PR to fix the conditional. Looks like the `if` clause needed to be under `build` and not `jobs`. The checks are running now.

**Comment by rsimha** [comment]
Native compiler build is green on all platforms for the November version of Closure. I'm merging this now, and will upgrade `amphtml` soon after.

**Pull request merged**
